identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -130,12 +132,15 @@ Insufficient.info.to.find.user=Provided information is insufficient to identify
 No.recovery.supported.claims.found=No recovery supported claims found
 Unknown.password.recovery.option=Unknown Password Recovery Option
 Username.cannot.be.empty=Username cannot be empty.
+Contact.cannot.be.empty=Contact cannot be empty.
+Invalid.contact=Invalid contact
 Account.confirmation.sent.to.your.email=Account confirmation has been sent to your email.
 Create.an.account=Create An Account
 Enter.required.fields.to.complete.registration=Enter required fields to complete registration of-
 Next=Next
 Already.have.an.account=Already have an account?
 Username.recovery.information.sent.to.your.email=Username recovery information has been sent to your email. Try again with different attributes if the email is not received.
+Username.recovery.information.sent.to.your.mobile=Username recovery information has been sent to your mobile. Try again with different attributes if the SMS is not received.
```

Comment:
`attributes` sounds a bit technical to an average end user, doesn't it? Better if we can use a common term.

Reply:
How about something like "Username recovery details have been sent to the mobile number provided. If the SMS is not received, try again using different information."

Reply:
Fixed from https://github.com/wso2/identity-apps/pull/7005/commits/851f59a741041758153a524cf49f5cf555bfe9bc